test/JDBC/upgrade/latest/schedule

```diff
@@ -466,6 +466,7 @@ print_null
 unquoted_string
 doublequoted_string
 operator_whitespace
+operator_atatvar
```

Comment:
For this Mvu test, we should put it into different upgrade test files

Reply:
there is no upgrade aspect since the things being fixed here all throw errors prior to the fix